runtime-builder/tests/structure/test_config.yaml

```diff
@@ -3,12 +3,10 @@ schemaVersion: "1.0.0"
 fileExistenceTests:
 - name: 'Go SDK'
   path: '/usr/local/go/'
-  isDirectory: true
```

Comment:
Is there similar checks implemented?

Reply:
structure test just knows whether it's looking for a directory of a file now, so the attribute was redundant.